lib/msf/core/auxiliary/password_cracker.rb

```diff
@@ -90,11 +90,10 @@ def new_password_cracker(cracking_application)
     rescue Metasploit::Framework::PasswordCracker::PasswordCrackerNotFoundError => e
       fail_with(Msf::Module::Failure::BadConfig, e.message)
     end
-    cracker_version = cracker.cracker_version
-    if cracking_application == 'john' && cracker_version.nil? || !cracker_version.include?('jumbo')
+    if cracker.cracker == 'john' && (cracker.cracker_version.nil? || !cracker.cracker_version.include?('jumbo'))
```

Comment:
```suggestion
    cracker_version = cracker.cracker_version
    if cracker.cracker == 'john' && (cracker_version.nil? || !cracker_version.include?('jumbo'))
```

Keep the local var, as `cracker.cracker_version` currently shells out each time it is called.